src/main/java/com/yelp/nrtsearch/server/grpc/NrtsearchServer.java

```diff
@@ -148,74 +150,77 @@ public NrtsearchServer(
 
   @VisibleForTesting
   public void start() throws IOException {
-    List<Plugin> plugins = pluginsService.loadPlugins();
-
-    LuceneServerImpl serverImpl =
-        new LuceneServerImpl(configuration, remoteBackend, prometheusRegistry, plugins);
-    GlobalState globalState = serverImpl.getGlobalState();
-
-    registerMetrics(globalState);
-
-    if (configuration.getMaxConcurrentCallsPerConnectionForReplication() != -1) {
-      replicationServer =
-          NettyServerBuilder.forPort(configuration.getReplicationPort())
-              .addService(
-                  new ReplicationServerImpl(
-                      globalState, configuration.getVerifyReplicationIndexId()))
-              .executor(
-                  ExecutorFactory.getInstance()
-                      .getExecutor(ExecutorFactory.ExecutorType.REPLICATIONSERVER))
-              .maxInboundMessageSize(MAX_MESSAGE_BYTES_SIZE)
-              .maxConcurrentCallsPerConnection(
-                  configuration.getMaxConcurrentCallsPerConnectionForReplication())
-              .maxConnectionAge(configuration.getMaxConnectionAgeForReplication(), TimeUnit.SECONDS)
-              .maxConnectionAgeGrace(
-                  configuration.getMaxConnectionAgeGraceForReplication(), TimeUnit.SECONDS)
-              .build()
-              .start();
-    } else {
-      replicationServer =
-          ServerBuilder.forPort(configuration.getReplicationPort())
-              .addService(
-                  new ReplicationServerImpl(
-                      globalState, configuration.getVerifyReplicationIndexId()))
-              .executor(
-                  ExecutorFactory.getInstance()
-                      .getExecutor(ExecutorFactory.ExecutorType.REPLICATIONSERVER))
+    try (Timer _timer = BootstrapMetrics.nrtsearchBootstrapTimer.startTimer()) {
```

Comment:
Let's just get the time manually to avoid multiple levels of indentation. It's also useful if we need to log the time.